### PR TITLE
Theme docs with shape

### PR DESCRIPTION
**Pull request by taylor-vann**
Heyyo!

Added an example of shape to the theming docs!

#2133

### CODE REVIEW

File: docs/theming.md [hunk 1]
```diff
@@ -217,18 +217,23 @@ html {
...
+  --mdc-shape-large: 16px;
+}
+```
+
 MWC doesn't support cut corners.
 
-See also: [Material Design shape](https://material.io/design/shape/applying-shape-to-ui.html#).
--->
+See also: [Material Design type system](https://material.io/design/shape/applying-shape-to-ui.html#shape-scheme)
+
```
Comment (asyncLiz) [inline docs/theming.md:236]: This should still be "shape" and missing a period. Also remove the extra newline so there's only one empty line between the sentence and the next section.

```suggestion
See also: [Material Design shape](https://material.io/design/shape/applying-shape-to-ui.html#shape-scheme).
```

**Review by asyncLiz — CHANGES_REQUESTED** [review]

File: docs/theming.md (hunk 1 above)
Comment (taylor-vann) [reply docs/theming.md:236]: gooood catch

**Review by taylor-vann — COMMENTED** [review]
Resolved in the following commit :)

File: docs/theming.md (hunk 1 above)
Comment (taylor-vann) [reply docs/theming.md]: resolved in the next commit

**Review by asyncLiz — APPROVED** [review]
LGTM

**Pull request merged**